### ISSUE TITLE
Idea: Panel view to manage all comments for the site

### ISSUE THREAD

**Issue by sebastiangreger** [sebastiangreger/kirby3-commentions#17]
A future [panel view](https://getkirby.com/docs/reference/plugins/extensions/panel-views) could serve as a centralized dashboard for managing all comments. This would be of particular value for webmentions, as some Indieweb users are happy to receive webmentions, but do not display them.

**Comment by sebastiangreger** [comment]
This shouldn't be too hard to do, reusing the panel components already customized for this. Yet, this is not so much a technical job, but a design task - postponing this to a future 1.1 version for the benefit of proper design work; the "inbox section" already serves the most important needs for basic use.

**Comment by fabianmichael** [comment]
This sounds really tempting, but I’m not sure if this offers any additional benefits in addition of a global commentions tab or section.

If we are going that route, it would make sense to have an additional widgets/"mini-section", intended for being added to `site.yml` (Dashboard), showing stats like incomming webmentions and whether the cron job did run smoothly.

**Comment by sebastiangreger** [comment]
Agreed. I believe that a tab on `site.yml` would be the ideal location for the "commentions dashboard" - it is actually a much more visible place (the panel views are rather hidden and maybe more suitable for settings or rarely used stuff than something that might require constant attention by the admin).

The one significant benefit of panel views is that they do not require blueprint changes: once the plugin is installed, the panel view is there, whereas the tab requires some setup - not a showstopper, though.

**Comment by fabianmichael** [comment]
> The one significant benefit of panel views is that they do not require blueprint changes: once the plugin is installed, the panel view is there, whereas the tab requires some setup - not a showstopper, though.

Agreed! Well, it won’t hurt to add the view in addition to the section (like you said, it’s probably pretty easy because we have almost everything in the section already). The dashboard widget/tab could be optional then. We could also provide a pre-defined blueprint for the commentions tab, so a user just needs to add this and don’t have to worry about picking the right section, icon etc.
